keyboards/ergodox_ez/rules.mk

```diff
@@ -31,11 +31,15 @@ SWAP_HANDS_ENABLE= yes # Allow swapping hands of keyboard
 SLEEP_LED_ENABLE = no
 API_SYSEX_ENABLE = no
 RGBLIGHT_ENABLE = yes
+RGBLIGHT_CUSTOM_DRIVER = yes
+
 RGB_MATRIX_ENABLE = no # enable later
 DEBOUNCE_TYPE = eager_pr
 
 # project specific files
-SRC += matrix.c
+SRC += matrix.c \
+       ws2812.c \
+	   led_bb_i2c.c
```

Comment:
Since we're not bitbanging anymore, I think this should probably be renamed.

Reply:
Well, technically, aren't bitbanging the WS2812 protocol data? 

But yeah, I can change the name

Reply:
I thought it was referring to the I2C. I'm not entirely sure what the setup is here, I'm guessing half the LED data are being bitbanged directly, and the other half are being sent over I2C to the other...half, right? Maybe it is still a fitting name. 🤷‍♂

Reply:
Well, somebody has a hacked a Atmega168p chip to be used instead of the normal chip that the ergodox ez uses.  

https://github.com/nicolai86/ErgoDox-EZ/tree/master/src

Specifically, it looks like the `led` array data is being sent over i2c, specifically
```
int main()
{
    TWI_Slave_Initialise(I2C_ADDR << 1);
    TWI_Start_Transceiver();
    DDRB |= _BV(ws2812_pin);
    sei();

    uint8_t rxMessage[TWI_BUFFER_SIZE];

    while(1) {
        if (!TWI_Transceiver_Busy() && TWI_statusReg.RxDataInBuf)
        {
            TWI_Get_Data_From_Transceiver(rxMessage, TWI_BUFFER_SIZE);
            ws2812_sendarray((uint8_t *)rxMessage, TWI_BUFFER_SIZE);
            TWI_Start_Transceiver();
        }
        
        if (TIFR1 & _BV(OCF1A))
        {
            TIFR1 = _BV(OCF1A);
        }
    }
}
```

So, I'm not really sure if bitbang is correct term here.